Source/JavaScriptCore/runtime/OptionsList.h

```diff
@@ -557,12 +557,8 @@ bool hasCapacityToUseLargeGigacage();
     v(Bool, verboseBBQJITAllocation, false, Normal, "Logs extra information about register allocation during BBQ JIT"_s) \
     v(Bool, verboseBBQJITInstructions, false, Normal, "Logs instruction information during BBQ JIT"_s) \
     v(Bool, disableBBQConsts, false, Normal, "Wasm <type>.const instructions in BBQ JIT won't lower to a const BBQ::Value"_s) \
-    v(Bool, useWasmLLInt, true, Normal, nullptr) \
```

Comment:
Won't you remove the `useWasmIPInt` option? I'm curious about what would happen if we run Wasm with `useWasmIPInt=false` after this change.

Reply:
Start using BBQ immediately.